### ISSUE TITLE
code error for running NB_nontarget_test_semseg.py 

### ISSUE THREAD

**Issue by hansoogithub** [C0ldstudy/PointSecGuard#1]
thank you for the project, but when i run the code

python NB_nontarget_test_semseg.py --data_path <data_path>
that is
python NB_nontarget_test_semseg.py --data_path data/s3dis

i get the following error
File "NB_nontarget_test_semseg.py", line 300, in <module>
    main(args)
File "NB_nontarget_test_semseg.py", line 102, in main
    model_name = os.listdir(expereiment_dir+'/logs')[0].split('.')[0]
IndexError: list index out of range

in my 
/data/s3dis
i have
Stanford3dDataset_v1.2_Aligned_Version 
and inside has 
Area_1 Area_2 until Area_6

please advise. thank you

**Comment by C0ldstudy** [comment]
Have you used the process [script](https://github.com/C0ldstudy/PointSecGuard/blob/main/PointNet/data_utils/collect_indoor3d_data.py) to convert the raw data to npy files?

**Comment by hansoogithub** [comment]
yes i used the script and it created the npy files as in this picture

![pointneterror](https://github.com/C0ldstudy/PointSecGuard/assets/71685330/11388c31-0f87-4084-b963-6166e69cded0)

**Comment by C0ldstudy** [comment]
In that case, I guess you need to use the training [script](https://github.com/C0ldstudy/PointSecGuard/blob/main/PointNet/train_semseg.py) to train a model at first?

**Comment by hansoogithub** [comment]
do you have a trained model i can download?

RuntimeError: cuDNN error: CUDNN_STATUS_EXECUTION_FAILED
